D3D12HelloWorld/Graphics/CommandList.cs

```diff
@@ -120,6 +123,27 @@ public sealed class CommandList : IDisposable {
             SetGraphicsRootDescriptorTable(rootParameterIndex, mShaderResourceViewDescriptorHeap, constantBufferView.CpuDescriptorHandle, 1);
         }
 
+        /// <summary>
+        /// Sets a constant buffer view (CBV) in the root signature for the graphics pipeline.
+        /// </summary>
+        /// <param name="rootParameterIndex">The index of the root parameter in the root signature.</param>
+        /// <param name="constantBufferView">The constant buffer view to set.</param>
+        /// <exception cref="InvalidOperationException">Thrown when the shader resource view descriptor heap is null.</exception>
+        public void SetGraphicsRootConstantBufferViewGpuBound(int rootParameterIndex, ConstantBufferView constantBufferView) {
```

Comment:
Exposed this version to allow the higher DX12GE abstraction to work with resources already uploaded to the GPU, to avoid the need to copy the descriptors.  PRobably should do this for all the `SetGraphicsRoot***` methods, or remove this again but add an optional boolean to each one to toggle this behaviour consistently across them all....?